src/sentry/static/sentry/app/components/dropdownButton.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import Button from './buttons/button';
+import InlineSvg from './inlineSvg';
+
+const DropdownButton = ({isOpen, children}) => {
+  return (
+    <StyledButton isOpen={isOpen}>
+      <StyledChevronDown />
```

Comment:
Should we have an option on chevron position? I think it should default to the right side

Reply:
rn it's kinda a la carte but we could make a chevron component that takes a direction

Reply:
Oops I meant in relation to `children`

Reply:
oooooh yeah that's a good idea